app/controllers/application_controller.rb

```diff
@@ -11,8 +11,7 @@ class ApplicationController < ActionController::Base
   include Vmdb::Logging
 
   if Vmdb::Application.config.action_controller.allow_forgery_protection
-    db = MiqDatabase.connected? ? MiqDatabase.first : nil
-    protect_from_forgery :secret => db ? db.csrf_secret_token : SecureRandom.hex(64), :except => :csp_report, :with => :exception
+    protect_from_forgery :secret => SecureRandom.hex(64), :except => :csp_report, :with => :exception
```

Comment:
Let's enable the magical before_action, `verify_authenticity_token`, but provide a random secret token.

We'll reset the secret in the Rails::Server proceses via `MiqUiWorker.preload_for_worker_role`.